sema/ro/getter/rogetter.py

```
@@ -0,0 +1,83 @@
+            uri = uri + "/"
+        self._uri = uri
+        self._output_path = Path(output_path or ".")
+        self._result = ROGetterResult()
+        self._force = force
+
+    @Trace.init(Trace)
+    def process(self) -> ROGetterResult:
+        query_result = (
+            Graph()
+            .parse(
+                data=requests.get(f"{self._uri}ro-crate-metadata.json").text,
```

Ruff (0.15.13)

[error] Probable use of `requests` call without timeout

(S113)